src/test/resources/mocks/et-daily-list/etDailyList.json

```diff
@@ -47,8 +47,7 @@
                               "individualForenames": "This is a forename",
```

Comment:
We should use the example JSON on the confluence page for these + the press list

Reply:
Sorry, I missed them. Both example JSON has been added.